### PR TITLE
Blog app - setup and controllers

### PR DESCRIPTION
**Pull request by roshan-bajgain**
## Blog app - setup and controllers
- Create a new app. Give it a meaningful name.
- Initialize your project with Git.
- Make sure that your project has Postgres database set up.
- For each URL you should create: action, view and route
- View files should include only placeholders written in plain HTML

### CODE REVIEW

**Comment by roshan-bajgain** [comment]
Hello, Reviewer I just repeat the module so I make small changes so You find only one commit. It was approved before thanks.

File: config/routes.rb [hunk 1]
```diff
@@ -8,5 +8,5 @@
   get "/users", to: "users#index"
   get "/users/:id/posts", to: "posts#index"
   get "/users/:id", to: "posts#show"
-  get "/users/:id/posts/:id" to: "posts#post"
+  get "/users/:id/posts/:id", to: "posts#post"
```
Comment (rindrajosia) [inline config/routes.rb]: [REQUIRED] Please make sure that all routes are present as you need it in the next milestone. For example:

- [x] posts#show should be /users/:user_id/posts/:id

- [x]  users#show  should be /users/:id

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues to revise before continuing to the next project but hang in there and read the comments below with care. You can do it! :muscle:

## To highlights 👍 

- No linter errors

## Required Changes ♻️
_Check the comments under the review._

- [x] Please make sure that app/views/users/show.html.erb is present

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by uwadonat — APPROVED** [review]
Hi **@[roshan-bajgain](https://github.com/roshan-bajgain),** 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉 
cong


# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**